index.js

```diff
@@ -132,13 +134,11 @@ function do_bundle(file, opts){
                     return jspm_opts;
         })();
 
-        var method = opts.selfExecutingBundle?'bundleSFX':'bundle';
-
+        var method = opts.selfExecutingBundle?'buildStatic':'bundle';
+        
         info_log('calling `jspm.'+method+"('"+jspm_input+"','"+jspm_output+"',"+JSON.stringify(jspm_opts)+');`', infos);
 
-        return Promise.resolve(
-            jspm[method](jspm_input, jspm_output, jspm_opts)
-        )
+        return builder[method](jspm_input, jspm_output, jspm_opts)
```

Comment:
I like `bluebird`'s long stack traces and that it doesn't swallow unhandled exceptions.
Would we get these two things if we use `jspm`'s promise?